mpu6050/registers.go

```diff
@@ -1,132 +1,65 @@
+func (p *Dev) write8(reg uint8, datum byte) error {
+	err := p.write(reg, []byte{datum})
+	return err
+}
 
-// Registers. Names, addresses and comments copied from the datasheet.
-const (
-	// Self test registers
-	SELF_TEST_X = 0x0D
-	SELF_TEST_Y = 0x0E
-	SELF_TEST_Z = 0x0F
-	SELF_TEST_A = 0x10
-
-	SMPLRT_DIV   = 0x19 // Sample rate divider
-	CONFIG       = 0x1A // Configuration
-	GYRO_CONFIG  = 0x1B // Gyroscope configuration
-	ACCEL_CONFIG = 0x1C // Accelerometer configuration
-	FIFO_EN      = 0x23 // FIFO enable
-
-	// I2C pass-through configuration
-	I2C_MST_CTRL   = 0x24
-	I2C_SLV0_ADDR  = 0x25
-	I2C_SLV0_REG   = 0x26
-	I2C_SLV0_CTRL  = 0x27
-	I2C_SLV1_ADDR  = 0x28
-	I2C_SLV1_REG   = 0x29
-	I2C_SLV1_CTRL  = 0x2A
-	I2C_SLV2_ADDR  = 0x2B
-	I2C_SLV2_REG   = 0x2C
-	I2C_SLV2_CTRL  = 0x2D
-	I2C_SLV3_ADDR  = 0x2E
-	I2C_SLV3_REG   = 0x2F
-	I2C_SLV3_CTRL  = 0x30
-	I2C_SLV4_ADDR  = 0x31
-	I2C_SLV4_REG   = 0x32
-	I2C_SLV4_DO    = 0x33
-	I2C_SLV4_CTRL  = 0x34
-	I2C_SLV4_DI    = 0x35
-	I2C_MST_STATUS = 0x36
-
-	// Interrupt configuration
-	INT_PIN_CFG = 0x37 // Interrupt pin/bypass enable configuration
-	INT_ENABLE  = 0x38 // Interrupt enable
-	INT_STATUS  = 0x3A // Interrupt status
-
-	// Accelerometer measurements
-	ACCEL_XOUT_H = 0x3B
-	ACCEL_XOUT_L = 0x3C
-	ACCEL_YOUT_H = 0x3D
-	ACCEL_YOUT_L = 0x3E
-	ACCEL_ZOUT_H = 0x3F
-	ACCEL_ZOUT_L = 0x40
+// write write a byte buffer to sequential incrementing adresses (1 byte per address) in individual
+// write operations.
+func (p *Dev) write(addr uint8, buff []byte) error {
+	p.conn.Tx(uint16(p.address), append([]byte{addr}, buff...), nil)
+	return nil
+}
 
-	// Temperature measurement
-	TEMP_OUT_H = 0x41
-	TEMP_OUT_L = 0x42
-
-	// Gyroscope measurements
-	GYRO_XOUT_H = 0x43
-	GYRO_XOUT_L = 0x44
-	GYRO_YOUT_H = 0x45
-	GYRO_YOUT_L = 0x46
-	GYRO_ZOUT_H = 0x47
-	GYRO_ZOUT_L = 0x48
-
-	// External sensor data
-	EXT_SENS_DATA_00 = 0x49
-	EXT_SENS_DATA_01 = 0x4A
-	EXT_SENS_DATA_02 = 0x4B
-	EXT_SENS_DATA_03 = 0x4C
-	EXT_SENS_DATA_04 = 0x4D
-	EXT_SENS_DATA_05 = 0x4E
-	EXT_SENS_DATA_06 = 0x4F
-	EXT_SENS_DATA_07 = 0x50
-	EXT_SENS_DATA_08 = 0x51
-	EXT_SENS_DATA_09 = 0x52
-	EXT_SENS_DATA_10 = 0x53
-	EXT_SENS_DATA_11 = 0x54
-	EXT_SENS_DATA_12 = 0x55
-	EXT_SENS_DATA_13 = 0x56
-	EXT_SENS_DATA_14 = 0x57
-	EXT_SENS_DATA_15 = 0x58
-	EXT_SENS_DATA_16 = 0x59
-	EXT_SENS_DATA_17 = 0x5A
-	EXT_SENS_DATA_18 = 0x5B
-	EXT_SENS_DATA_19 = 0x5C
-	EXT_SENS_DATA_20 = 0x5D
-	EXT_SENS_DATA_21 = 0x5E
-	EXT_SENS_DATA_22 = 0x5F
-	EXT_SENS_DATA_23 = 0x60
-
-	// I2C peripheral data out
-	I2C_PER0_DO      = 0x63
-	I2C_PER1_DO      = 0x64
-	I2C_PER2_DO      = 0x65
-	I2C_PER3_DO      = 0x66
-	I2C_MST_DELAY_CT = 0x67
+// MPU 6050 REGISTER ADRESSES
+const (
+	SMPRT_DIV    uint8 = 0x19
+	CONFIG       uint8 = 0x1A
+	GYRO_CONFIG  uint8 = 0x1B
+	ACCEL_CONFIG uint8 = 0x1C
+	FIFO_EN      uint8 = 0x23
+	I2C_MST_CTRL uint8 = 0x24
+	PWR_MGMT_1   uint8 = 0x6B
+	ACCEL_XOUT_H uint8 = 0x3B
+	TEMP_OUT_H   uint8 = 0x41
+	GYRO_XOUT_H  uint8 = 0x43
+)
```

Comment:
This is entirely optional, but:
I think we should change these constants to be unexported. There is no reason why a user of the package would need these, and if they do that's a leaky abstration.
Most drivers do currently export these values, but I think it would be a good idea to unexport these.